config/router-chatbot.js

```diff
@@ -91,7 +88,9 @@ router.post('/', (req, res) => {
     })
     .then(msg => {
       controller.debug(req, `sendMessage:${msg}`);
-      mobilecommons.chatbot(req.body, mobilecommonsOptinPath, msg);
+      controller.setCurrentCampaign(req.user, req.campaign_id);
+
+      mobilecommons.chatbot(req.body, process.env.CAMPAIGNBOT_MOBILECOMMONS_OIP, msg);
```

Comment:
Ah, ok. We just need to be sure it'll throw some kind of error if the value is undefined.